pyproject.toml

```diff
@@ -94,3 +96,27 @@ markers = [
 
 [tool.flit.module]
 name = "astro"  # Or "astro.sql" if you just want this directory, not the entire 'astro'.
+
+[tool.mypy]
+color_output = true
+#disallow_any_generics = true
+#disallow_incomplete_defs = true
+#disallow_untyped_defs = true
+#explicit_package_bases = true
+# TODO: Once we complete #281 we can replace the files parameter by the mypy_path
+files = [
+    "src/astro/settings.py",
```

Comment:
I was worried about this, glad you found a way to restrict mypy to files we are currently working on.